grails-app/controllers/org/pih/warehouse/product/ProductSupplierController.groovy

```diff
@@ -155,12 +158,15 @@ class ProductSupplierController {
                     defaultPreference.productSupplier = productSupplierInstance
                     productSupplierInstance.addToProductSupplierPreferences(defaultPreference)
                 }
+            } else {
+                if (defaultPreference) {
+                    productSupplierInstance.removeFromProductSupplierPreferences(defaultPreference)
+                    defaultPreference.delete()
```

Comment:
Should we allow removing global preference types?

Reply:
Yes. It's not required anywhere.